packages/graphql/src/services/sessions.ts

```diff
@@ -1,13 +1,132 @@
-import { SessionStatus } from '@klicker-uzh/prisma'
+import {
+  Question,
+  QuestionType,
+  SessionBlockStatus,
+  SessionStatus,
+} from '@klicker-uzh/prisma'
 import { ContextWithUser } from '@lib/context'
-import { getRedis } from '../lib/redis'
 
-export async function startSession(
-  { id }: { id: string },
+function processQuestionData(question: Question): AllQuestionTypeData {
+  switch (question.type) {
+    case QuestionType.SC:
+    case QuestionType.MC: {
+      return {
+        ...question,
+        options: question.options!.valueOf(),
```

Comment:
! is a declaration like ? in TS, that tells the compiler the value will always be there (so no "might be undefined" warnings). Use carefully (i.e., never).